README.md

```diff
@@ -27,15 +27,35 @@ BAR Lobby is a new WIP lobby for the RTS game [Beyond All Reason](https://www.be
 
 ### Recommended Environment
 
-It is highly recommended to use [VSCode](https://code.visualstudio.com/) for development, as it provides full, built-in TypeScript support, as well as useful extensions such as [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) and [Volar](https://marketplace.visualstudio.com/items?itemName=johnsoncodehk.volar).
+It is highly recommended to use [VSCode](https://code.visualstudio.com/) for development, as it provides full, built-in TypeScript support, as well as useful extensions such as [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) and [Vue - Official (previously Volar)](https://marketplace.visualstudio.com/items?itemName=johnsoncodehk.volar).
 
 ### Requirements
 
--   [Node.js](https://nodejs.org/en/download/)
+-   [Node.js 18](https://nodejs.org/en/download/)
 
-### Project setup:
+#### Installing Node 18 on Bash
+```bash
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.40.0/install.sh | bash
```

Comment:
I would just document what you need to have node 18, and for *unix* base systems we recommend nvm, and link to that docs.

I would definitely not copy nvm install instruction, pinned to specific version.